SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -174,7 +174,8 @@ if [ -n "${BUILDER_VERSION}" ]; then
 else
     _log "Creating environment ${ENV_NAME} using ${BUILDER}..."
 fi
-source "${BUILDER_PATH}"
+# To avoid accessing EOS and bigger runtimes, configure HOME to be the user's home directory
```

Comment:
I would change this to:
```
# To prevent builders (e.g. mamba) from caching files on EOS, which slows down the creation of the environment,
# configure HOME to be the user's local directory
```